sql

```diff
@@ -0,0 +1,46 @@
+create table restoraunt_info(id serial,
+	                         r_name varchar(200) not null,
+							 delivery char(1) not null,
```

Comment:
change name to "has_delivery" and change type to boolean

Reply:
Fixed